query/function.go

```diff
@@ -141,7 +153,6 @@ func MakeTransformMetricFunction(name string, parameterCount int, transformer fu
 		Name:        name,
 		MinArgument: parameterCount + 1,
 		MaxArgument: parameterCount + 1,
-		Groups:      true,
```

Comment:
This was accidentally added to `Transform` instead of `Aggregate`, making it possible to use `group by` in transforms but not in aggregations.